product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "Preparing your Azure account"
+
+---
+
+BigAnimal requires you to configure your Azure subscription, before you deploy your clusters. The configurations that you perform ensure that your Azure subscription is prepared to meet your clusters' resource requirements, such as: 
+
+ * Are the necessary Azure Service Providers registered for your subscription?
+ * Is there a restriction on SKUs for the Standard Esv3 and Standard Dv2 VM size families?
+ * Is there a sufficient limit on the number vCPU or Public IP addresses in your region?
+  
+!!! Note
+    Before proceeding, see [Understanding requirements in Azure](../01_preparing_azure/01_understanding_qotas_in_azure) for information about planning for your clusters' resource requirements in Azure.
+
+## Step 1: Check for requirements
+
+EDB recommends using the `azure-preflight` script to check whether all resource requirements are met in your subscription. However, you can also manually check the requirements using the Azure CLI or the Azure Portal.
+
+ * [Method 1: Use EDB's shell script](#use-edbs-shell-script) (recommended)
+ * [Method 2: Manually check requirements](#manually-check-requirements)
+
+
+### Use EDB's shell script
+
+EDB provides a shell script called [`azure-preflight`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/azure-preflight.sh), which automatically checks whether resource requirements are met in your Azure subscription based on the clusters you plan to deploy.
+
+1. Open the [Azure Cloud Shell](https://shell.azure.com/) in your browser.
+ 
+1. From the Azure Cloud Shell, use the following command to run the `azure-preflight.sh` script.  
+
+ ```
+ curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/azure-preflight.sh | bash -s -- --location <azure_region> --pgtype eNsv3 --endpoint private --ha --with-infra
+ ```
+
+1. Run the `azure-preflight.sh` script with the following options: 
+   - `<azure_region>` is the Azure region you plan to deploy your clusters.
+   - `<pg_type>` **e*N*s_v3** indicates the type of virtual machine instance using *N* cores.
+   - `<end_point>` indicates either **public** or **private** endpoints.
+   - `--ha` indicates that the instance is run with high availability enabled.
+   - `--with-infra` indicates that no clusters have been created in the region as yet.
+  
+  
+ For example, if you want to deploy a cluster in the `eastus2` region with `e2s_v3` cores, in a `public` network, and no existing cluster, run the following command:
```

Comment:
For example, if you want to deploy a cluster in the Azure subscription `12412ab3d-1515-2217-96f5-0338184fcc04` of `eastus2` region with `e2s_v3` cores, in a `public` network, and no existing cluster, run the following command:

Reply:
Replaced.